installers/rpm-code/codes/srw.sh

```diff
@@ -1,4 +1,23 @@
 #!/bin/bash
+: to_install_at_nersc <<EOF
+# Hinsts for installing SRW at NERSC
+
+## Install python
+curl radia.run | bash -s nersc-pyenv
+
+## Load fftw with:
+`module load  cray-fftw`
```

Comment:
remove backticks, because that means something in shell which I don't think you want to do.

Reply:
4ef9378